cloud_experts_tutorials/cloud-experts-gitops-with-argocd.adoc

```diff
@@ -0,0 +1,181 @@
+:_mod-docs-content-type: ASSEMBLY
+[id=“cloud-experts-gitops-with-argocd]
+= Tutorial: Demonstrate GitOps on Managed OpenShift with ArgoCD
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: cloud-experts-gitops-with-argocd
+
+toc::[]
+
+//Mobb content metadata
+//Brought into ROSA product docs 2023-10-31
+// ---
+// date: '2021-06-17'
+// title: Demonstrate GitOps on Managed OpenShift with ArgoCD
+// aliases: ['/experts/demos/gitops/']
+// tags: ["GitOps", "ROSA", "ARO"]
+// authors:
+//   - Steve Mirman
+//   - Connor Wooley
+// ---
+
+The purpose of this document is to help you get  OpenShift GitOps running in your cluster, including deploying a sample application and demonstrating how ArgoCD ensures environment consistency.
+
+[NOTE]
+====
+If you prefer visual guide, you can watch a quick start video on link:https://www.youtube.com/watch?v=Gi18iemF1yI[YouTube].
+====
+
+== Prerequisites
+
+* You have a Managed OpenShift Cluster with cluster-admin rights.
+* You have the following tools installed:
+    ** link:https://git-scm.com/download/[git]
```

Comment:
```suggestion
** link:https://git-scm.com/download/[Git]
```
I think you can delete the spaces before the bullets (https://docs.asciidoctor.org/asciidoc/latest/lists/unordered/).